docs/pages/dev-guide/developer-guide.md

```diff
@@ -1002,7 +1002,17 @@ plans:
         tasks: [sidecar]
 ```
 
-To initiate this plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`. Its progress can be monitored like any other plan: by issuing GET requests against the `/v1/plans/side-car-example` endpoint.
+Note that the command definition for the sidecar task includes an environment variable, `PLAN_PARAMETER`, that is not defined elsewhere in the service definition. This parameter can be supplied by the user at the time that the plan is initiated, with that parameter being propagated to the environment of every task launched by the plan. To initiate the plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`, with the header `Content-Type: application/json` set and with a JSON body consisting of environment variable name/value pairs as in the following:
```

Comment:
Suggest:

The command definition for the sidecar task includes an environment variable, `PLAN_PARAMETER`, that is not defined elsewhere in the service definition. You can supply this parameter when the plan is initiated. The parameter will be propagated to the environment of every task launched by the plan. 

To initiate the plan, execute an HTTP POST request against the `/v1/plans/sidecar-example/start` endpoint with the header `Content-Type: application/json` and a JSON body consisting of environment variable name/value pairs. For example: